### DISCUSSION TITLE
[Suggestion] Make console commands and outputs translatable strings

### DISCUSSION THREAD

**Discussion by unbeatable-101** [OpenTTD/OpenTTD#9250] [Features]
As of now they are just English strings, they should be converted to translatable ones like the interface of OpenTTD is.

https://github.com/OpenTTD/OpenTTD/pull/9203#issuecomment-834782205

**Comment by auge8472** [comment]
I don't like this idea – *at least* for the commands. Whenever someone thinks, a translation of a command is not well enough, one would change the translation and with it the notation of the (tanslated) command itself. I don't know how often this would happen but I would not want to struggle with changing notation of commands.

On the other side, with commands only in English language command output in other languages would be nonsense (*in my humble opinion*).

**Comment by embeddedt** [comment]
-1 for translatable command *names*, not because I don't like accessibility, but because it will introduce complexity with any type of scripting that relies on commands. I don't know if this is possible clientside but it should be possible serverside. It also means that tutorials would have to include multiple variants of the same command.

However, I wouldn't object to having the output be translated - this makes sense and matches how programming works in general (keywords in English, but the final program interacts with the user in the desired language).

**Comment by extrazi** [comment]
when it comes to translating the descriptions of the commands in the console, this might be the option
